### PR TITLE
Installation path

### PR DESCRIPTION
**Pull request by fippli**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -36,33 +34,45 @@ check_permissions () {
...
   # If a second argument is passed
   # clone the url and enter into the
   # cloned project assuming it follows
   # the correct climate structure.
   if [ ! $# -eq 0 ]; then
-    clear_cloned
+    clear_clones
     clone_cli_repository "${1}"
     go_to_cloned_repo "$(get_repository_name "${1}")"
+    pwd
```
Comment (Jim-Holmstroem) [inline install.sh]: is this debug stuff?

File: utils.sh [hunk 2]
```diff
@@ -63,19 +67,18 @@ get_repository_name () {
...
 
 go_to_cloned_repo () {
-  cd "${package_cloned}/${1}" || exit
+  cd "${package_clones_path}/${1}" || exit
 }
 
-clear_cloned () {
-  rm -rf "${package_cloned:?}"/*
+clear_clones () {
+  rm -rf "${package_clones_path:?}"/*
```
Comment (Jim-Holmstroem) [inline utils.sh]: shouldn't be an issue in modern GNU system, but you never know, I would double check that package_clones_path is nonempty before proceeding

File: install.sh (hunk 1 above)
Comment (fippli) [reply install.sh]: yes

File: utils.sh (hunk 2 above)
Comment (fippli) [reply utils.sh]: good idea

Comment (fippli) [reply utils.sh]: it is exported in main though, might still be a problem?

**Pull request merged**